IdentityCore/src/util/ios/UIApplication+MSIDExtensions.h

```diff
@@ -21,6 +21,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#import <UIKit/UIKit.h>
```

Comment:
isn't this imported in pch?

Reply:
Not on Mac...:

```
#if TARGET_OS_IPHONE
#import <UIKit/UIKit.h>
#else
#import <AppKit/AppKit.h>
#endif
```

Reply:
well, this file shouldn't be imported on mac at all and you shouldn't need to add UIKit import here:)

Reply:
good point...